sharded_queue/drivers/fifottl.lua

```diff
@@ -388,7 +388,28 @@ function method.peek(args)
 end
 
 function method.release(args)
-    local task = box.space[args.tube_name]:update(args.task_id, { {'=', index.status, state.READY} })
+    local delay = args.delay or 0
+    local ttl = args.ttl or args.options.ttl or time.MAX_TIMEOUT
+    local ttr = args.ttr or args.options.ttr or ttl
+    local priority = args.priority or args.options.priority or 0
+    local next_event
+    local status = state.READY
+
+    if delay > 0 then
+        status = state.DELAYED
+        ttl = ttl + delay
+        next_event = time.event(delay)
+    else
+        next_event = time.event(ttl)
```

Comment:
Don't get this: do we refresh the `ttl` on each `release` call? Will it break the current behavior?